superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberViz.tsx

```diff
@@ -311,7 +291,7 @@ class BigNumberVis extends React.PureComponent<BigNumberVisProps> {
     return (
       <div className={className} style={{ height }}>
         {this.renderFallbackWarning()}
-        {this.renderKicker(kickerFontSize * height)}
+        {this.renderKicker((kickerFontSize || 0) * height)}
```

Comment:
lots of minor checks that needed to be added when the interfaces/types where updated